configs/policy.yaml

```diff
@@ -1979,4 +1979,91 @@ acct_settings:
         'Sebastopol': 10
         'Sonoma': 10
         'Unincorporated Sonoma': 5
-        'Windsor': 3
\ No newline at end of file
+        'Windsor': 3
+
+
+unit_preservation:
+    enable_in_scenarios: ["23"]
+    geography: county_name
+    settings: # change geography as needed
+      'Alameda':
+        first_unit_filter: (sesit_id == 'dr' or sesit_id == 'hradr') and tra_id == tra_id
```

Comment:
I don't quite understand what "tra_id == tra_id" or "gg_id == gg_id" does, it is always true, isn't it?

Reply:
I needed to check whether values existed in those columns or not, with Pandas query. And apparently you can't use "is not null/NaN/None", so this was a suggested way to do the query that checks for a value.

Reply:
That's a weird one.  I didn't know that one but I guess it makes sense.

Reply:
Oh I just did something like `gg_id > ''` to check if it is empty.

Reply:
Yeah that didn't work in this case. Both `' '` and `value` are not ideal criteria. 

Here is the background on the use of `value`: https://stackoverflow.com/questions/26535563/querying-for-nan-and-other-names-in-pandas
"In general, you could use @local_variable_name, so something like would work, but nan isn't equal to itself, so value == NaN will always be false. One way to hack around this is to use that fact, and use value != value as an isnan check. We have"